VK_V1.0/VK_V1.0/NewsScene/Model/News.swift

```diff
@@ -0,0 +1,15 @@
+// News.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Модель новости
```

Comment:
```suggestion
/// Новости
```
Не пиши это слово! Модель (я и так понимаю что это)